extensions/objects/templates/svg-filename-editor.component.spec.ts

```diff
@@ -222,36 +230,39 @@ describe('SvgFilenameEditor', function() {
     };
     svgFilenameCtrl.fillPicker = mockPicker;
     svgFilenameCtrl.strokePicker = mockPicker;
+    svgFilenameCtrl.bgPicker = mockPicker;
   }));
 
   it('should update diagram size', function() {
     var WIDTH = 100;
     var HEIGHT = 100;
-    var MAX_DIAGRAM_WIDTH = 491;
-    var MAX_DIAGRAM_HEIGHT = 551;
     svgFilenameCtrl.diagramWidth = WIDTH;
     svgFilenameCtrl.diagramHeight = HEIGHT;
     svgFilenameCtrl.onWidthInputBlur();
     expect(svgFilenameCtrl.currentDiagramWidth).toBe(WIDTH);
     svgFilenameCtrl.onHeightInputBlur();
     expect(svgFilenameCtrl.currentDiagramHeight).toBe(HEIGHT);
-    svgFilenameCtrl.diagramWidth = 600;
-    svgFilenameCtrl.diagramHeight = 600;
-    svgFilenameCtrl.onWidthInputBlur();
-    expect(svgFilenameCtrl.currentDiagramWidth).toBe(MAX_DIAGRAM_WIDTH);
-    svgFilenameCtrl.onHeightInputBlur();
-    expect(svgFilenameCtrl.currentDiagramHeight).toBe(MAX_DIAGRAM_HEIGHT);
   });
 
-  it('should return information on diagram size', function() {
-    var maxDiagramWidth = 491;
-    var maxDiagramHeight = 551;
-    var helpText = (
-      'This diagram has a maximum dimension of ' +
-      maxDiagramWidth + 'px X ' + maxDiagramHeight +
-      'px to ensure that it fits in the card.');
-    expect(svgFilenameCtrl.getDiagramSizeInfo()).toBe(helpText);
-  });
+  it('should reset to maximum and minimum dimension limits correctly',
+    function() {
+      var MAX_DIAGRAM_WIDTH = 490;
+      var MAX_DIAGRAM_HEIGHT = 550;
```

Comment:
This might be the maximum possible, but do you think it would be sensible to restrict it to something smaller and less random than these 2 numbers? Also, instead of having these constants in the spec file, should they be constants defined globally if it is also used in the code?

Reply:
@nithusha21  I restricted it to the maximum so that the users would have the flexibility to save it till that maximum. So if I reduce it, the users might not be able to create a larger image which might be possible. These constants are only used in the svg editor, so if required should I move them to the constants.ts file?

Reply:
Usually you have a svg-filename-editor.constants.ts right?

Reply:
So should I be reducing the size and if so what is the suggested size?

Reply:
The reduction is optional, not something I would be able to suggest right off. Depends on an end user. From my experience with uploading stuff on other websites... I would prefer if the size was fixed to something like 250px x 250px rather than have 2 random limits for the size. But that is an optional change..

The other one seems more important... If we ever want to resize the limits, it should be possible to do it from 1 location in the codebase. If these numbers are used elsewhere too, then it is not the best to redeclare it here. Save it in a constants file (not the global one, but probably something within the component), and reuse that.

Reply:
Done.